glusterd2/commands/volumes/volume-create.go

```diff
@@ -18,8 +20,20 @@ import (
 
 const (
 	maxMetadataSizeLimit = 4096
+	minVolumeSize        = 10
```

Comment:
question:
want to understand why minimum volume size is 10

Reply:
I think we may have to increase this upto 20, this is because `mkfs.xfs` was not working on smaller size.

For now this is moved from `plugin/smartvol` code to here.